far/FarEng.hlf.m4

```diff
@@ -3225,58 +3225,52 @@ string is changed to that of the found string. For example:
     └────────────────┴────────────────────┴──────────────────────┘
 
 @FindAllMenu
-$ #Редактор: меню с результатами поиска всех вхождений#
-    В этом меню доступны следующие сочетания клавиш:
+$ #Editor: All matching entries menu#
+  The following key combination are available in this menu:
 
-      #F5#                   - переключить размер меню
+    #F5#              - Toggle menu size.
 
-      #Ctrl+Up, Ctrl+Down#   - скроллировать текст редактора
+    #Ctrl+Up#         - Scroll the text in the editor.
+    #Ctrl+Down#
 
-      #Ctrl+Enter#           - перейти к позиции найденного текста
+    #Ctrl+Enter#      - Go to the position of the found text.
+    #Ctrl+Left#
+      #mouse click#
 
-      #Add#                  - добавить стековую закладку с текущей позицией
+    #Gray +#          - ^<wrap>Add stack bookmark with the current position.
 
-      #ПравыйCtrl+0..9#      - установить закладку 0..9 в текущей позиции
-
-      #Ctrl+Shift+0..9#      - установить закладку 0..9 в текущей позиции
-
-      #ЛевыйCtrl+0..9#       - перейти к позиции закладки 0..9
+    #RightCtrl+0..9#  - Set bookmark 0..9 at the current position.
+    #Ctrl+Shift+0..9# 
 
+    #LeftCtrl+0..9#   - Go to the bookmark 0..9.
 
 @FileOpenCreate
 $ #Editor: Open/Create file#
-    With #Shift+F4#, one can open the existing file or create a new file.
-
-    Если файл с заданным именем не найден, то создаётся новый.
-По умолчанию используется кодовая страница ANSI, кодовую страницу по умолчанию можно изменить
-в ~настройках редактора~@EditorSettings@).
-При необходимости из #списка# можно выбрать другую кодовую страницу.
-
-    При открытии существующего файла кодовая страница выбирается
-по ряду правил (см. Примечания).
-При необходимости требуемую кодовую страницу можно задать явно, выбрав её из #списка#.
-Там же доступно и "~Автоматическое определение~@CodePageAuto@".
-
-    Примечания.
 
-    Если при открытии существующего файла не задавать кодовую страницу явно (активен пункт "По умолчанию"),
-то кодовая страница будет выбрана исходя из следующих правил:
+    The #Shift+F4# key combination opens an existing file or creates
+a new file.
 
-    1) Если файл открывался ранее, то возможно для него уже сохранена информация о
-кодовой странице (зависит от состояния параметра "[x] Сохранять позицию файла"
-в ~настройках редактора~@EditorSettings@).
-В этом случае используется запомненная ранее кодовая страница.
+    If the specified file does not exist, a new file will be created.
+The code page for the new file is specified in the #Code page# list.
+By #default#, the ANSI code page is used. The default code page can be
```

Comment:
Maybe skip this sentence, given that the default code page can be something else than ANSI?

Reply:
Good point. I'll reword. To be sure: default code page is defined in the global editor setting dialog; no additional logic is applied, right?

@zg0 Why? Could you please be more specific?